README.md

```diff
@@ -85,3 +85,6 @@ Branches exist on your local and on the remote. Always delete your branch in bot
 - $ `git checkout main`
 - $ `git pull origin main`
 - $ `git branch -d <branch-name>`
+
+
+Person first_name:string last_name:string phone_number:string
```

Comment:
Please move this into your markdown file

Reply:
copy